db_s3_backup.py

```diff
@@ -69,54 +69,55 @@ def create_mysql_dump(mysql_config, s3_bucket, s3_bucket_key_name, filepath, ver
 
 	f.close()
 
-# 
+#
 # Create SQLite dump
-# 
+#
 
 def create_sqlite_dump(sqlite_config, s3_bucket, s3_bucket_key_name, filepath, verbose = False):
 	if verbose:
 		print('Copying SQLite file to {filepath}'.format(filepath=filepath))
-	
+
 	copyfile(sqlite_config['NAME'], filepath)
-	
+
 	# Send file to S3
 	f=open(filepath, 'r')
 	upload_dump_s3(f, s3_bucket, s3_bucket_key_name, verbose)
 	f.close()
 
-# 
+#
 # Upload dump to Amazon S3
-# 
+#
 
 def upload_dump_s3(f, s3_bucket, s3_bucket_key_name, verbose = False):
-	s3_file = Key(s3_bucket, name=s3_bucket_key_name)
+	s3_file = Key(s3_bucket, name= 'dumps/' + s3_bucket_key_name)
 
 	if verbose:
 		print('Uploading dump to Amazon S3: {url}'.format(url=s3_file.generate_url(expires_in=0, query_auth=False)))
 
+	f.seek(0)
```

Comment:
sure thing, `seek(0)` won't hurt, but I'm surprised it didn't work before since when we open a new file descriptor with `open(filename, 'r')` it's supposed to have the cursor at the beginning of the file

anyway, it's good you've addressed that!